portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsSingleSelectionField.testcase

```diff
@@ -762,6 +762,72 @@ definition {
 			key_fieldReferenceValue = "SingleSelection1234");
 	}
 
+	@description = "LPS-90435 This is a test verify that the Single selection Form field with non-english site default can not be broken"
+	@priority = "4"
+	test SingleSelectionFormFieldWithNonenglishSiteDefaultCanNotBeBroken {
+		property test.name.skip.portal.instance = "FormsSingleSelectionField#SingleSelectionFormFieldWithNonenglishSiteDefaultCanNotBeBroken";
+
+		task ("Issue 1:The the Single selection Form field with non-english site default can be broken when change the site default language after create a form.") {
+			JSONGroup.addGroup(groupName = "Site Name 1");
+
+			ApplicationsMenu.gotoSite(site = "Site Name 1");
+
+			FormsAdmin.addForm(groupName = "Site Name 1");
+
+			Form.editName(formName = "Form 1");
+
+			Form.editDescription(formDescription = "description 1");
+
+			Form.gotoAddField(
+				fieldPositionNumber = "0",
+				fieldType = "Single Selection");
+
+			FormFields.addSingleSelectionOption(
+				optionName = "Option 1",
+				optionValue = "1");
+
+			FormFields.addSingleSelectionOption(
+				optionName = "Option 2",
+				optionValue = "2");
+
+			Form.save();
+
+			ProductMenu.changeDefaultLanguage(defaultPortalLanguage = "German (Germany)");
+
+			ProductMenu.gotoPortlet(
+				category = "Content & Data",
+				portlet = "Forms");
+
+			FormsAdminNavigator.gotoEditFormByName(formName = "Form 1");
+
+			Form.editName(formName = "New Form 1");
+		}
+
+		task ("Issue 2:The the Single selection Form field with non-english site default can be broken when create a form after change the site default language.") {
+			JSONGroup.addGroup(groupName = "Site Name 2");
+
+			ApplicationsMenu.gotoSite(site = "Site Name 2");
+
+			ProductMenu.changeDefaultLanguage(defaultPortalLanguage = "German (Germany)");
+
+			FormsAdmin.addForm(groupName = "Site Name 2");
+
+			Form.gotoAddField(
+				fieldPositionNumber = "0",
+				fieldType = "Single Selection");
+
+			FormFields.addSingleSelectionOption(
+				optionName = "Option 1",
+				optionValue = "1");
+
+			FormFields.addSingleSelectionOption(
+				optionName = "Option 2",
+				optionValue = "2");
+
+			Form.save();
```

Comment:
After this last line in the second test, we want to confirm that the field name and options were added by going back into the Form and viewing the field name and options - this is the assertion that fulfills the purpose of this test.